packages/stack/blockchain/src/index.js

```diff
@@ -109,6 +110,13 @@ export default class Blockchain {
     });
     this.blockchainApi.registerAPIs("ethereum");
     this.blockchainApi.registerRequests("ethereum");
+
+    if (this.blockchainConfig.enabled && this.blockchainConfig.client === "geth") {
+      warnIfPackageNotDefinedLocally("embark-geth", this.embark.logger.warn.bind(this.embark.logger));
+    }
+    if (this.blockchainConfig.enabled && this.blockchainConfig.client === "parity") {
```

Comment:
We have constants for "geth" and "parity" in `constants.blockchain.clients` (please double-check the spelling)